src/AppBundle/Resources/config/services.yml

```diff
@@ -26,7 +26,8 @@ services:
 
     # auto-register all commands as services
     AppBundle\Command\:
-        resource: '../../Command'
+        resource: '../../Command/*'
+        tags: ['console.command']
```

Comment:
@dvesh3 Would that be also possible (fully supported by Symfony 4?) for the core commands, instead of defining every single command? 
https://github.com/pimcore/pimcore/blob/master/bundles/CoreBundle/Resources/config/commands.yml

Reply:
IMO, It should work! ;)

Reply:
@dvesh3 then this one should be a quite easy one :) https://github.com/pimcore/pimcore/issues/3799